Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -135,6 +160,18 @@ - (void) SetPlaneDetectionEnabled:(bool)enabled {
     planeDetectionEnabled = enabled;
 }
 
+- (bool) TrySetMeshDetectorEnabled:(const bool)enabled {
+    meshDetectionEnabled = enabled;
+    return true;
```

Comment:
It seems like this needs to do a runtime check. My understanding is mesh detection is only available on iOS 13.4 or higher, so I would think this would be:
```suggestion
if (@available(iOS 13.4, *)) {
    meshDetectionEnabled = enabled;
    return true;
}

return false;
```

Reply:
I also realized that mesh detection would only be available if scene reconstruction is supported on the device, so I went ahead and added that check in as well. In the case that they enable mesh detection but device doesn't support it, should return false.